signal-hook-mio/src/lib.rs

```diff
@@ -0,0 +1,258 @@
+#![doc(
+    html_root_url = "https://docs.rs/signal-hook-mio/0.1.0/",
+    test(attr(deny(warnings)))
+)]
+#![warn(missing_docs)]
+//! A crate offering integration with the MIO runtime.
+//!
+//! There are different sub modules for supporting different MIO
+//! versions. The support for a version must be activated by a
+//! feature flag:
+//!
+//! * `support-0_6` for sub module [`v0_6`]
+//! * `support-0_7` for sub module [`v0_7`]
```

Comment:
I wonder if the feature should be named `support-*` when the module is `v*`. Would it be possible to unify both on `v*`?